build/includes/sdk.mk

```diff
@@ -20,14 +20,15 @@
 #   \__, |_| \_\_|    \____|   |_|\___/ \___/|_|_|_| |_|\__, |
 #   |___/                                               |___/
 
-build_sdk_base_version = $(call sha,$(build_path)/build-sdk-images/tool/base/Dockerfile)
+grpc_release_tag = v1.57.0
+
+build_sdk_base_version = $(call sha,$(build_path)/build-sdk-images/tool/base/Dockerfile)_$(grpc_release_tag)
```

Comment:
This now forces a rebuild when the grpc version is updated. Before this wasn't happening.